index.rst

```diff
@@ -6,24 +6,219 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
-
 Abstract
 ========
 
-This is a technote for the description of results from Individual Hardpoint Breakaway Test. 
+This technote describes the analysis and results derived from individual hard point breakaway tests conducted with the M1M3 surrogate.
+The notebook is located within `notebooks_vandv <https://github.com/lsst-sitcom/notebooks_vandv/blob/tickets/SITCOM-838/notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-838_Anaysis.ipynb>`_ GitHub repository.
+With all the steps guarded with reasonable timeouts, so problems are detected if hard point cannot travel to reach low or high limit switches, etc.
+**If this test shows that the hard points do not work properly at the limits, this could be one of the blockers for the installation of M1M3 until it is solved.** 
+
+
+Hardpoint Breakaway Test
+========================
+
+The active support system of the M1M3 includes six axial hard point actutators in the hexapod configuration. :cite:`2018SPIE10700E..3GD`
+These hard point actuators should minimize forces during slews at any TMA position and be kept under the breakaway limit. 
+The breakaway limits for each hard points should happen in the range of -4420 to -3456 N for retraction and 2981 to 3959 N for extension.
+The followings steps are performed during the individual hard point breakaway test.
+
+1. Move hard point in negative (increasing tension) direction until either a low limit switch is actuated
+
+2. Move hard point in positive (increasing compression) direction till the high limit switch is triggered
+
+3. Move hard point downwards (increasing tension) until the low limit switch is hit
+
+4. Move hard point back to reference position
+
+5. Wait for the hard point to reach the reference position
+
+
+Note that for tests only full travel, from low limit to high limit and back, performed in steps 2 and 3 shall be considered. Other hard point movements do not guarantee a full hard point motion range will be examined.
+
+
```

Comment:
Can you add a new section here with a list of requirements that we are trying to verify here? If you don't have the list right away, you can create a session and create a todo box as a reminder to do this in a new PR. Something like

```
Requirements and Tickets
===================

Find here the JIRA tickets and requirements associated with this test. 

.. todo::

    Add associated requirements to this technote.
```

Reply:
If you get the requirements quickly enough, simply add them. But consider adding them as part of a new ticket/PR.

Reply:
I would guess this would be found in section 3.7 of https://docushare.lsst.org/docushare/dsweb/Get/LTS-88. 
For example, 

3.7.7 Hardpoint Axial Breakaway Repeatability
ID: LTS-88-REQ-0024
Specification: The hardpoint axial breakaway SHALL have a repeatability after recovering
from either a compression or tension breakaway of 0.65 μm.

Let me add it on the "To do list" but definitely add it soon (just not today).